lib/miq_expression/field.rb

```diff
@@ -2,14 +2,19 @@ class MiqExpression::Field
   FIELD_REGEX = /
 (?<model_name>([[:upper:]][[:alnum:]]*(::)?)+)
 \.?(?<associations>[a-z_\.]+)*
--(?<column>[a-z]+(_[[:alnum:]]+)*)
+-
```

Comment:
Not necessary for this PR, but if this regex is getting increasingly complex it may be better to implement a bona fide parser. User feedback is valuable - LMK if you have any opinions either way